_topic_maps/_topic_map_rosa.yml

```diff
@@ -216,6 +216,10 @@ Topics:
   File: rosa-hcp-creating-cluster-with-aws-kms-key
 - Name: Creating a private cluster on ROSA with HCP
   File: rosa-hcp-aws-private-creating-cluster
+- Name: Creating a break glass credential for a ROSA with HCP cluster
+  File: rosa-hcp-sts-create-breakglass-cred
```

Comment:
Both of these entries are modules. If you want to do this, you need to reference the directory

Reply:
Thanks @EricPonvelle - it's more of a placeholder right now as they will probably fit into your external auth assembly...we shall see!